src/kvaser_interface.cpp

```diff
@@ -208,14 +210,17 @@ ReturnStatuses KvaserCan::registerReadCallback(std::function<void(void)> callabl
   else
   {
     readFunc = callable;
-    auto ret = KvaserReadCbProxy::registerCb(this, handle);
+    auto stat = canSetNotify(*(handle), &proxyCallback, canNOTIFY_RX, static_cast<void*>(this));
 
-    if (ret != ReturnStatuses::OK)
+    if (stat != canOK)
```

Comment:
Why did `ReturnStatuses::OK` change to `canOK`?

Reply:
It's a misleading diff. `stat` is returned by a call to the `canlib` API, which returns a `canlib` error code. The previous call was a call to a `kvaser_interface` API, which returns a `kvaser_interface` error code.